### PR TITLE
KOGITO-6978: Update insecure dependencies

### PR DESCRIPTION
**Pull request by thiagoelg**
# JIRA Issue
https://issues.redhat.com/browse/KOGITO-6978

# In this PR
Updated dependencies to fix security issues pointed out by dependabot.

The updated packages were:
- plist (npm)
- ch.qos.logback (mvn) on both dashbuilder/appformer and stunner-editors
- moment (npm)
- jboss.remoting (mvn) on serverless-workflow-diagram-editor
- jsoup (mvn) on serverless-workflow-diagram-editor
- node-forge (npm)
- simple-get (npm)
- trim-newlines (npm)
- ansi-regex (npm)
- mermaid (npm)

I ran the sanity checks as described in the documents and found no regressions with these updates, but please let me know if I missed anything.

### CODE REVIEW

File: packages/stunner-editors/pom.xml [hunk 1]
```diff
@@ -265,7 +265,7 @@
     <version.org.kogito.gwt-jsonix-schema-compiler>1.3.0</version.org.kogito.gwt-jsonix-schema-compiler>
 
     <!-- Third party Libraries -->
-    <version.ch.qos.logback>1.1.3</version.ch.qos.logback> <!-- Newer version in kie-parent causes ServiceLoader error -->
+    <version.ch.qos.logback>1.2.11</version.ch.qos.logback> <!-- Newer version in kie-parent causes ServiceLoader error -->
     <version.com.google.elemental2>1.1.0</version.com.google.elemental2>
     <version.com.google.guava>30.1-jre</version.com.google.guava>
     <version.com.google.gwt>2.9.0</version.com.google.gwt>
```
Comment (yesamer) [inline packages/stunner-editors/pom.xml:271]: Hi @thiagoelg Can you please update `jsoup` version in this file too? I think we should keep these two pom synchronized, when possible. Thx!

File: packages/serverless-workflow-diagram-editor/pom.xml [hunk 2]
```diff
@@ -261,7 +261,7 @@
     <version.org.kogito.gwt-jsonix-schema-compiler>1.3.0</version.org.kogito.gwt-jsonix-schema-compiler>
 
     <!-- Third party Libraries -->
-    <version.ch.qos.logback>1.1.3</version.ch.qos.logback> <!-- Newer version in kie-parent causes ServiceLoader error -->
+    <version.ch.qos.logback>1.2.11</version.ch.qos.logback> <!-- Newer version in kie-parent causes ServiceLoader error -->
```
Comment (yesamer) [inline packages/serverless-workflow-diagram-editor/pom.xml]: @thiagoelg Can you please double-check that comment? If obsolete can you please remove it?

File: packages/stunner-editors/pom.xml [hunk 3]
```diff
@@ -265,7 +265,7 @@
     <version.org.kogito.gwt-jsonix-schema-compiler>1.3.0</version.org.kogito.gwt-jsonix-schema-compiler>
 
     <!-- Third party Libraries -->
-    <version.ch.qos.logback>1.1.3</version.ch.qos.logback> <!-- Newer version in kie-parent causes ServiceLoader error -->
+    <version.ch.qos.logback>1.2.11</version.ch.qos.logback> <!-- Newer version in kie-parent causes ServiceLoader error -->
```
Comment (yesamer) [inline packages/stunner-editors/pom.xml]: @thiagoelg Can you please double-check that comment? If obsolete can you please remove it?

**Review by yesamer — COMMENTED** [review]
@thiagoelg Thank you for your PR!

**Review by caponetto — APPROVED** [review]
Thanks, @thiagoelg!!

File: packages/stunner-editors/pom.xml (hunk 3 above)
Comment (thiagoelg) [reply packages/stunner-editors/pom.xml]: Yep, these can be removed, thanks!

**Review by yesamer — APPROVED** [review]
Thank you @thiagoelg

**Review by tiagobento — APPROVED** [review]

**Pull request merged**